test-case/check-userspace-cardinfo.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+##
+## Case Name: check-userspace-cardinfo
+## Preconditions:
+##    N/A
+## Description:
+##    Get the card name from pactl info
+##    check if there is an available card to do the following tests
+## Case step:
+##    1. run pactl to get the cards info
+##    2. check the card name
+## Expect result:
+##    There is at least one available card for test
+##
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+func_opt_parse_option "$@"
+
+OLDIFS=$IFS
+cardlist=$(pactl list cards short)
+((available_card=0))
+IFS=$'\n'
+for card in $cardlist; do
+    dlogi "found card: $(echo $card|awk '{print $2}')"
```

Comment:
Also it would be better to have some output example here. If the output format changed, we can easily debug and update our code here.

Reply:
@xiulipan OK, I will add a comment for the output example